### ISSUE TITLE
CPA doesn't let me choose properties file

### ISSUE THREAD

**Issue by dlogan** [CellProfiler/CellProfiler-Analyst#251]
(Possibly related to #239 ?)
Using CPA developer's version from master branch on Mac, I launch CPA but the dialog flashes up and then immediately another dialog says this:
<img width="532" alt="screen shot 2017-08-24 at 1 31 34 pm" src="https://user-images.githubusercontent.com/1225595/29679781-91f10e5e-88d0-11e7-8460-dfb7933e2df7.png">
The problem is that I never get a chance to choose a properties file.
 
Maybe it has to do with my installation?  I recently installed **CellProfiler** using the conda installation instructions (https://github.com/CellProfiler/CellProfiler/wiki/Conda-Installation) which works, so I thought I'd give it a shot for CPA (since I'm not sure I have ever gotten CPA dev on my Mac in recent years).  The conda installation worked, but I needed to manually pip install a few packages (verlib, seaborn, sklearn -- I think that's it).

The console output looks normal afaik, like this:
```
(cellprofiler-analyst) [logand**@****:~/CellProfiler_Analyst_conda/CellProfiler-Analyst]$ pythonw ./CellProfiler-Analyst.py
INFO:root:No PIL.ArgImagePlugin
ERROR:root:CellProfiler Analyst requires a properties file. Exiting.
```

My java is updated and I (re)installed the JDK from the CP download site today.

Here is the output of pip freeze, if you think I may be missing something:
``` 
$ pip freeze
alabaster==0.7.10
appdirs==1.4.3
appnope==0.1.0
Babel==2.4.0
backports-abc==0.5
backports.shutil-get-terminal-size==1.0.0
backports.ssl-match-hostname==3.5.0.1
bleach==1.5.0
cellh5==1.3.0
centrosome==1.0.8
certifi==2016.2.28
configparser==3.5.0
contextlib2==0.5.5
cycler==0.10.0
Cython==0.26
decorator==4.1.2
docutils==0.14
entrypoints==0.2.3
enum34==1.1.6
funcsigs==1.0.2
functools32==3.2.3.post2
future==0.16.0
h5py==2.7.0
hmmlearn==0.2.0
html5lib==0.9999999
imagesize==0.7.1
inflect==0.2.5
ipykernel==4.6.1
ipython==5.3.0
ipython-genutils==0.2.0
ipywidgets==6.0.0
javabridge==1.0.14
Jinja2==2.9.6
jsonschema==2.6.0
jupyter==1.0.0
jupyter-client==5.1.0
jupyter-console==5.2.0
jupyter-core==4.3.0
lxml==3.8.0
mahotas==1.4.3
MarkupSafe==1.0
matplotlib==2.0.2
mistune==0.7.4
mock==2.0.0
MySQL-python==1.2.5
nbconvert==5.2.1
nbformat==4.4.0
networkx==1.11
notebook==5.0.0
numpy==1.13.1
olefile==0.44
packaging==16.8
pandas==0.20.3
pandocfilters==1.4.1
pathlib2==2.3.0
pbr==3.1.1
pexpect==4.2.1
pickleshare==0.7.4
Pillow==4.2.1
prokaryote==2.1.2
prompt-toolkit==1.0.15
ptyprocess==0.5.2
Pygments==2.2.0
pyparsing==2.2.0
python-bioformats==1.2.0
python-dateutil==2.6.1
pytz==2017.2
PyWavelets==0.5.2
pyzmq==16.0.2
qtconsole==4.3.1
raven==6.1.0
requests==2.14.2
scandir==1.5
scikit-image==0.13.0
scikit-learn==0.19.0
scipy==0.19.1
seaborn==0.8
simplegeneric==0.8.1
singledispatch==3.4.0.3
six==1.10.0
sklearn==0.0
snowballstemmer==1.2.1
Sphinx==1.6.3
sphinxcontrib-websupport==1.0.1
subprocess32==3.2.7
terminado==0.6
testpath==0.3
tifffile==0.9.0
tornado==4.5.1
traitlets==4.3.2
typing==3.6.2
verlib==0.1
wcwidth==0.1.7
widgetsnbextension==2.0.0
wxPython==3.0.0.0
wxPython-common==3.0.0.0
```
Thanks for any advice!

**Comment by jhung0** [comment]
Weird. I have the same versions as you, and it's fine. Have you tried
without conda?

**Comment by karhohs** [comment]
@dlogan I know wxPython in conda can cause trouble with CP. The workaround is to use a specific version `wxpython=3.0.2.0` from the `bjornfjohansson` channel. I have a copy of the environment file you can see [here](https://github.com/karhohs/conda-env-archetypes/blob/master/cellprofiler/mac/environment.yml)

**Comment by dlogan** [comment]
Thanks @jhung0 and @karhohs .  I had seen that note about the wxpython version and I was blindly hoping that I wouldn't have to worry since I didn't know how to implement that version immediately ;) .  So thanks for the env file, I will try it!

**Comment by dlogan** [comment]
As suggested, I updated my environment.yml file with the new wxpython=3.0.2.0 and get this error now:
```
(cellprofiler-analyst) [logand05@AMRL2RK0CSFVH8:~/CellProfiler_Analyst_conda/CellProfiler-Analyst]$ pythonw CellProfiler-Analyst.py
Traceback (most recent call last):
  File "CellProfiler-Analyst.py", line 22, in <module>
    import cpa.helpmenu
  File "/Users/logand05/CellProfiler_Analyst_conda/CellProfiler-Analyst/cpa/helpmenu.py", line 1, in <module>
    import wx
ImportError: No module named wx
```

This shows that the intended wx is loaded:
```
(cellprofiler-analyst) [logan***@****:~/CellProfiler_Analyst_conda/CellProfiler-Analyst]$ pip freeze | grep wx
wxPython==3.0.2.0
wxPython-common==3.0.2.0
```
Thoughts?

**Comment by jhung0** [comment]
Hmm... were you able to figure it out? I don't think I even have wxPython installed.

**Comment by DavidStirling** [comment]
Should be fixed in 3.0
